applications/DeepSpeed-Chat/training/utils/data/raw_datasets.py

```diff
@@ -642,6 +642,43 @@ def get_prompt_and_rejected(self, sample):
         )
         return None
 
+# Pleisto Dataset
+class PleistoYurenDataset(PromptRawDataset):
+
+    def __init__(self, output_path, seed, local_rank):
+        super().__init__(output_path, seed, local_rank)
+        self.dataset_name = "pleisto/yuren"
+        self.dataset_name_clean = "pleisto_yuren"
+        self.raw_datasets = load_dataset("pleisto/yuren")
+
+    def get_train_data(self):
+        return self.raw_datasets["train"]
+    
+    def get_eval_data(self):
+        return self.raw_datasets["validation"]
+    
+    def get_prompt(self, sample):
+        pos = sample['input'].find('[|AI|]')
+        return sample['input'][:pos+6]
+    
+    def get_chosen(self, sample):
+        pos = sample['input'].find('[|AI|]')
+        return sample['input'][pos:]
+    
+    def get_prompt_and_chosen(self, sample):
+        return sample['input']
```

Comment:
In order for us to accept any new dataset, you would need to follow the format defined in the PromptRawDataset class comments. This is to ensure format consistency when blending multiple datasets. For your dataset, at least I see following format issues that need to be fixed (you can check existing datasets in this file about ways to fix format):

(1) Need to remove "[|System|]The conversation between human and AI assistant. "
(2) Replace all "[|Human|]" with "Human: "
(3) Replace all "[|AI|]" with "Assistant: "
(4) Remove the " [|Human|]" at the end of each input string.